src/orders/views/OrderDetails/OrderUnconfirmedDetails/index.tsx

```diff
@@ -98,6 +109,26 @@ export const OrderUnconfirmedDetails: React.FC<OrderUnconfirmedDetailsProps> = (
       first: 30
     }
   });
+
+  const {
+    data: customerAddresses,
+    loading: customerAddressesLoading
+  } = useCustomerAddressesQuery({
+    variables: {
+      id: order?.user?.id
+    },
+    skip: !order?.user?.id || !isAnyAddressEditModalOpen(params.action)
+  });
+
+  const handleCustomerChangeAddresses = async (
+    data: Partial<OrderCustomerAddressesEditDialogOutput>
+  ): Promise<any> => {
+    await orderUpdate.mutate({
+      id,
+      input: data
+    });
+  };
```

Comment:
It's not a very good pattern. Here you just await promise, just to wrap it into another promise. Effectively comes to this:

```suggestion
  const handleCustomerChangeAddresses = (
    data: Partial<OrderCustomerAddressesEditDialogOutput>
  ): Promise<any> => orderUpdate.mutate({
      id,
      input: data
    });
```